lib/galaxy_test/base/populators.py

```diff
@@ -323,7 +323,7 @@ def _run_cwl_workflow_job(
             # workflows as well, and then make it the default. Or decide they are safe.
             "allow_tool_state_corrections": True,
         }
-        invocation_id = self.workflow_populator.invoke_workflow(
```

Comment:
Thanks for the fix, sorry about this.

Reply:
No worries, this is tested only in the CWL branch.